### PR TITLE
New Style: Gradient Dark 

### PR DESCRIPTION
**Pull request by samiaab1990**
Hi there, 
I created a new style "Gradient Dark". Here are some examples of what it looks like.
_I made some changes to my original request because of mistake duplicate classes_

**CSS**
GradientDarkCodeChunk

 **Javascript**
GradientDarkCodeChunkJavaScript

### CODE REVIEW

**Comment by yyyc514** [comment]
Removing your editor specific config file.  This is what `.editorconfig` is used for.

File: CHANGES.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ New languages:
   none.
 
 New styles:
-  none.
+  -*Gradient Dark* by [Samia Ali][]
```
Comment (yyyc514) [inline CHANGES.md]: This needs to be under 9.17 not 9.16... and you also need to add the link to the footnotes (see the other examples).

Comment (yyyc514) [reply CHANGES.md]: Needs a space after `-` for bulleted list.

Comment (samiaab1990) [reply CHANGES.md]: Thank you! I wasn't sure which version to put it under

Comment (yyyc514) [reply CHANGES.md]: Waiting for you to push an update.

**Comment by yyyc514** [comment]
@samiaab1990 Thanks for contributing!

**Pull request merged**